### DISCUSSION TITLE
PHP Error Log showing Unable to load dynamic library 'clos_ssa.so' when loading PHPMailer

### DISCUSSION THREAD

**Discussion by wtaulu** [PHPMailer/PHPMailer#2919] [Q&A]
My PHP Error Log keep showing this error everytime the PHPMailer is called.
This error seems to get thrown during the PHP Initialization.
`PHP Warning:  PHP Startup: Unable to load dynamic library 'clos_ssa.so' (tried: /opt/alt/php82/usr/lib64/php/modules/clos_ssa.so (/opt/alt/php82/usr/lib64/php/modules/clos_ssa.so: cannot open shared object file: No such file or directory), /opt/alt/php82/usr/lib64/php/modules/clos_ssa.so.so (/opt/alt/php82/usr/lib64/php/modules/clos_ssa.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0`

The PHPMailer seems to be working just fine and is sending out emails as I want, it is just my log getting filled up with these errors that I would like to clean up.

PHPMailer: 6.8.0 
PHP: PHP 8.2 (alt-php82)

Code Snippet:

```
use PHPMailer\PHPMailer\PHPMailer;
use PHPMailer\PHPMailer\Exception;
require_once 'PHPEmailer/Exception.php';
require_once 'PHPEmailer/PHPMailer.php';
require_once 'PHPEmailer/SMTP.php

$mail = new PHPMailer(true);
    try {
        $mail->SMTPDebug = 0; 
        $mail->isSMTP();
        $mail->Host       = "mail.xxxxx.com";
        $mail->SMTPAuth   = true;
        $mail->SMTPSecure = false;
        $mail->SMTPAutoTLS = false;
        $mail->Port       = 26;
        $mail->Username   = "xxxxxxxx";
        $mail->Password   = "xxxxxxxxxxx";
        $mail->From       = "xxxxxxxxxxxxxxxxx";
        $mail->FromName   = "xxxxxxxxx";
        $mail->addAddress('email@email.com');
        $mail->Subject = 'Subject';
        $mail->Body = 'Email Body';
        $mail->isHTML(true);
        $mail->send();
    } catch (phpmailerException $e) {
        $errmsg = $e->errorMessage();
    }
```
Fairly standard email setup.   As I mentioned, the emails are sending just fine but the error logs report that error everytime the email code is run.

**Comment by Synchro** [comment]
This has nothing to do with PHPMailer; it's a PHP configuration problem that will show up in *any* PHP script that you run. It means that your php.ini asks to load a PHP extension from a file called `clos_ssa.so`, but that file doesn't exist. Either install the missing extension, or disable the directive to load it in your php.ini. I have no idea what the `clos_ssa` extension does, though it seems to be something to do with Cloud Linux.
